src/development/going-live.md

```diff
@@ -1,103 +1,89 @@
 ```
 
-> **note**
-> You can test those routes on your development environments with:
-> * `http://[branch]-[project-id].[region].platform.sh`
-> * `http://www-[branch]-[project-id].[region].platform.sh`
+That will result in two domains being created on Platform.sh: `master-def456-abc123.us.platform.sh` and `www---master-def456-abc123.us.platform.sh`.  The former will automatically redirect to the latter.  In the `routes.yaml` file, `{default}` will automatically be replaced with `master-def456-abc123.us.platform.sh`.  In domain prefixes (like `www`), the `.` will be replaced with `---`.
 
-### HTTPS
+### Set your domain
 
-If you have uploaded your SSL certificate and you want to serve your
-entire site with HTTPS, here is what your routes would look like:
+Now, add a single domain to your Platform.sh project for `mysite.com`.  As soon as you do, Platform.sh will no longer serve `master-def456-abc123.us.platform.sh` at all.  Instead, `{default}` in `routes.yaml` will be replaced with `mysite.com` anywhere it appears when generating routes to respond to.
 
-```yaml
-"https://{default}/":
-    type: upstream
-    upstream: "app:http"
+### Configure your DNS provider
 
-"http://{default}/":
-    type: redirect
-    to: "https://{default}/"
-```
+On your DNS provider, you would create two CNAMEs:
 
-### Wildcard domains
+`mysite.com` should be a CNAME to `www---master-def456-abc123.us.platform.sh`.
+`www.mysite.com` should be a CNAME to `www---master-def456-abc123.us.platform.sh`.
 
-To configure a wildcard domain (*.mydomain.com):
+(Yes, both point to the same place.)
 
-- Add your domain to your project (in form of mydomain.com).
-- Add a route to your master branch serving `http://*.mydomain.com` with the upstream app:http.
+### Result
+
+Now, an incoming request for `mysite.com` will result in the following:
 
+1) Your browser asks the DNS network for `mysite.com`'s DNS record.  It responds with "it's an alias for `www---master-def456-abc123.us.platform.sh`".
+2) your browser asks the DNS network for `www---master-def456-abc123.us.platform.sh`'s DNS record.  It responds with "that's IP address 1.2.3.4".  (Or whatever the actual address is.)
+3) Your browser sends a request to `1.2.3.4` for domain `mysite.com`.
+4) Your router responds with an HTTP 301 redirect to `www.mysite.com`.
+5) Your browser looks up `www.mysite.com` and, as above, gets an alias for `www---master-def456-abc123.us.platform.sh`, which is IP 1.2.3.4.
+6) Your browser sends a request to `1.2.3.4` for domain `www.mysite.com`.  Your router passes the request through to your application which in turn responds with whatever it's supposed to do.
 
-## 3 - DNS
+On subsequent requests, your browser will know to simply connect to `1.2.3.4` for domain `www.mysite.com` and skip the rest.  The entire process takes only a few milliseconds.
 
-Configure your DNS provider to point your domain to your
-[Platform.sh](https://platform.sh) Master environment.
+## Old Stuff, is it relevant?
```

Comment:
Anything past here in this page I am not sure if we should keep or reword.  My registrar lets me CNAME to foo.com just fine.  Also, my read of this section is "you can't actually serve foo.com from us even though all of our documentation says you can, because reasons."  Which... seems both incorrect and not a good thing to say. :smile: If kept it would still need to be greatly rewritten.